go.mod

```diff
@@ -13,7 +13,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-module github.com/apache/spark-connect-go/v34
+module github.com/apache/spark-connect-go/v1
```

Comment:
Is this entry point which defines the version for this go library?

Reply:
I guess this `v34` is accumulated since this go library began. Is it the same as the release version? I'm not sure if it is okay to change it like this.

Reply:
Previously we use `v34` to make it align with Spark 3.4 at that time. It is not accumulated version number. Things will still work after changing `v34` to `v1`.